frontend/src/containers/AddChart.tsx

```diff
@@ -55,6 +56,7 @@ function AddChart() {
   const [title, setTitle] = useState("");
   const [summary, setSummary] = useState("");
   const [chartType, setChartType] = useState<ChartType>(ChartType.LineChart);
+  const [showTitle, setShowTitle] = useState(true);
```

Comment:
Instead of having this state variable and the `handleShowTitleChange` function. You can simply get the value of the checkbox by using the `getValues()` function that you get from the `useForm<FormValues>()` hook in Line 41. .

Reply:
So, in Line 530 for example, you can do something like this instead: 

```
<LineChartPreview
  title={getValues().showTitle ? title : ""}
/>
```

Reply:
For some reason, implementing this change results in the preview not updating appropriately. The preview doesn't update when the checkbox is toggled, only when the summary or title is being edited.

Reply:
I see. Ok that's fine then.